### ISSUE TITLE
Keras Models Don't Behave Deterministically with Fixed Seed

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#345]
Models which use Keras under the hood don't seem to behave deterministically with fixed seed. Such models include the graph-convolutional models and the low-data models. Here are some potential fixes

- For now, we can just report the numbers in the benchmark table with a * attached to indicate the randomness.
- Long run fix is probably to remove Keras dependency so we can control sources of randomness as with the custom Tensorflow models (#296)

@miaecle Have I summarized the issue correctly?

**Comment by miaecle** [comment]
@rbharath Thanks for setting up this issue, this is exactly what I have observed on the results.

**Comment by rbharath** [comment]
Related to #292
